test/integration/connect/test_connect.py

```diff
@@ -359,6 +375,38 @@ def test_connect_and_get_commands_errors(tt_cmd, tmpdir_with_cfg):
         stop_app(tt_cmd, tmpdir, "test_app")
 
 
+def test_connect_and_execute_quit(tt_cmd, tmpdir_with_cfg):
```

Comment:
I'm not quite sure if this test should be in the same commit as the empty crash fix. Maybe it could be moved  to a separate commit. Up to you.

Reply:
It seems to me that everything is fine here. It is possible to split reproducer & fix it, but here it is a simple fix and a simple reproducer. Split will only complicate things.